requirements-dev.txt

```diff
@@ -3,4 +3,10 @@ coverage
 flake8
 mock
 nose
-pytest==3.5.1
+pytest
+retrying
+dcos-shakedown
+cryptography
+https://github.com/dcos/dcos-e2e/archive/2018.12.10.0.zip
+https://github.com/dcos/dcos-test-utils/archive/master.zip
+https://github.com/adamtheturtle/dcos-installer-tools/archive/master.zip
```

Comment:
I am not comfortable long term with having my name on this if it is being used by another team. That means that we should move it to the Mesosphere org IMO. I will make an issue for doing that and we can update this when appropriate.

Reply:
In the mean time I can import it from the vendored version in dcos-e2e if you'd rather that as well.

Reply:
No - keep it as is 😄

Reply:
switched to the mesosphere team url.